packages/benchmarks/src/pipelines/discoveryEvalToBraintrust.ts

```diff
@@ -0,0 +1,46 @@
+import { runPipeline } from "mongodb-chatbot-evaluation";
+import configConstructor from "../discovery.config";
+import { radiantModels } from "../radiantModels";
+import { strict as assert } from "assert";
+import { evaluateRegexMatch } from "../braintrust/evaluateRegexMatch";
+import { assertEnvVars } from "mongodb-rag-core";
+import { envVars } from "../envVars";
+runPipeline({
+  configConstructor,
+  pipeline: async (generate, evaluate, report, config) => {
+    const { BRAINTRUST_API_KEY } = assertEnvVars({ BRAINTRUST_API_KEY: "" });
+    const timestamp = Date.now();
+    const evalReports: { [label: string]: Record<string, unknown> } = {};
+    // Run different model evals in parallel
+    await Promise.allSettled(
+      radiantModels.map(async (model) => {
+        const name = `${model.label}_discovery_conversations`;
+        const {
+          commandRunMetadata: { _id: genRunId },
+        } = await generate(name);
+        const {
+          commandRunMetadata: { _id: qualityEvalRunId },
+        } = await evaluate("mentions_mongodb", genRunId);
+        const { report: reported } = await report(name, qualityEvalRunId);
+        evalReports[model.label] = reported.data;
+        // Put the evaluations in Braintrust
```

Comment:
This seems like it should be wrapped up as a `BraintrustReporter` interface or something that we define in the config and call via `report()` vs grabbing things off the config (though that's still useful as an escape hatch)

Reply:
agree. making separate ticket b.c i think to do this elegantly w/in the evaluation package will require some thoughtfulness and not trivial code additions. 

ticket -